pkg/webhook/controlplane/ensurer_test.go

```diff
@@ -629,10 +568,8 @@ func checkKubeAPIServerDeployment(dep *appsv1.Deployment, k8sVersion string) {
 	switch {
 	case k8sVersionAtLeast131:
 		Expect(c.Command).NotTo(ContainElement(HavePrefix("--feature-gates")))
-	case k8sVersionAtLeast128:
+	default:
 		Expect(c.Command).To(ContainElement("--feature-gates=InTreePluginGCEUnregister=true"))
-	default: // = 1.27
-		Expect(c.Command).To(ContainElement("--feature-gates=CSIMigrationGCE=true,InTreePluginGCEUnregister=true"))
 	}
```

Comment:
This could be now replaced with an if else statement. It might be acceptable to have a switch case for 3 conditions. IMO it is not justified for only 2 conditions. A simple if else would do the job.